src/OscapScannerRemoteSsh.cpp

```diff
@@ -121,8 +121,9 @@ QStringList OscapScannerRemoteSsh::getCommandLineArgs() const
     }
     else
     {
+        QString userTailoringFile = mSession->getUserTailoringFilePath();
         args += buildEvaluationArgs(mSession->getOpenedFilePath(),
-            mSession->hasTailoring() ? mSession->getTailoringFilePath() : QString(),
+            mSession->hasTailoring() ?  ( userTailoringFile.isEmpty() ? mSession->getTailoringFilePath() : userTailoringFile ): QString(),
```

Comment:
Thank you for the suggestion. I followed the idea of a wrap method and ended up changing the use of getUserTailoringFilePath().